h/static/scripts/tests/controllers/confirm-submit-controller-test.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+var ConfirmSubmitController = require('../../controllers/confirm-submit-controller');
+var util = require('./util');
+
+describe('ConfirmSubmitController', function () {
+
+  var ctrl;
+
+  afterEach(function () {
+    if (ctrl) {
+      ctrl.element.remove();
+      ctrl = null;
+    }
+  });
+
+  /**
+   * Make a <button type="submit"> with the confirm submit controller
+   * enhancement applied and return the various parts of the component.
+   *
+   */
+  function component(windowConfirmReturnValue) {
```

Comment:
Protip (no change required here): You can use destructuring as a slightly nicer way to do boolean parameters in ES6: `function component({windowConfirmReturnValue})`. Usage: `component({windowConfirmReturnValue: true})`. This is mostly useful if you have several such parameters.